engine/apps/email/alert_rendering.py

```diff
@@ -50,6 +50,6 @@ def build_subject_and_message(alert_group, emails_left):
     )
 
     title = str_or_backup(templated_alert.title, title_fallback)
-    subject = f"[{title}] You are invited to check an alert group"
+    subject = f"[{title}] You are invited to check an alert group".strip()
```

Comment:
I think we should do `.replace("\n", "")` instead of `.strip()` on the subject, otherwise newlines could still make it into the email subject.

Reply:
@vadimkerr 
You are right, 
will change it. 
BTW a ``trim`` in the notification template seems fixed the problem I have experienced. 
Thanks